src/Sarif.Viewer.VisualStudio/Sarif/CodeFlow.extensions.cs

```diff
@@ -38,5 +34,17 @@ public static AnnotatedCodeLocationCollection ToAnnotatedCodeLocationCollection(
 
             return model;
         }
+
+        public static CallTree ToCallTree(this CodeFlow codeFlow)
+        {
+            if (codeFlow == null || codeFlow.Locations.Count == 0)
```

Comment:
Absolutely, I'll change it to use the null conditional operator.

The spec says that `locations` is required and must have at least one element. I think we need to be resilient against invalid SARIF, though.